code/game/objects/machinery/sentries.dm

```diff
@@ -228,7 +228,8 @@
 	req_one_access = list(ACCESS_MARINE_ENGINEERING, ACCESS_MARINE_ENGPREP, ACCESS_MARINE_LEADER)
 	hud_possible = list(MACHINE_HEALTH_HUD, SENTRY_AMMO_HUD)
 	var/turret_flags = TURRET_HAS_CAMERA|TURRET_SAFETY|TURRET_ALERTS
-	var/list/iff_signal = list(ACCESS_IFF_MARINE)
+	/// The faction used to determine friendlies from hostiles
+	var/faction
```

Comment:
I prefer this to be like it was in the comments on the iff signal, a bitfiel so you can have multiple iff signals

Reply:
i mean this was never used, and i don't think anyone plan to use it?

I certainly don't need it for hvh mode

Reply:
> 
> 
> i mean this was never used, and i don't think anyone plan to use it? And you mean a list, access are not bitfields
> 
> I certainly don't need it for hvh mode

the comment said it was to be turned into a bitfield later